frontend/public/components/monitoring.tsx

```diff
@@ -940,17 +949,14 @@ class SilenceForm_ extends React.Component<SilenceFormProps, SilenceFormState> {
               </label>
             </div>
             <div className="col-xs-1">
-              <button type="button" className="btn btn-link" onClick={() => this.removeMatcher(i)} aria-label="Remove matcher">
-                <i className="fa fa-minus-circle" aria-hidden="true" />
+              <button type="button" className="btn btn-link btn-link--inherit-color" onClick={() => this.removeMatcher(i)} aria-label="Remove matcher">
+                <MinusCircleIcon />
               </button>
             </div>
           </div>)}
-          <Button
-            onClick={this.addMatcher}
-            type="button"
-            variant="link">
-            <PlusCircleIcon /> Add More
-          </Button>
+          <button type="button" className="btn btn-link btn--silence-add-more" onClick={this.addMatcher}>
```

Comment:
Looks like we've changed from `Button` to `button`. Was that intentional?

Reply:
Good catch.  That was a rebasing error I've fixed.